### PR TITLE
documentation: Update to READMEs: Add a monthly download badge

### PR DESCRIPTION
**Pull request by chrfoyer**
Why?
Allow users to get a brief overview of popularity of the project or plugin

What?
A monthly download badge on each Readme.

How has this been tested?
Each badge has been checked on GitHub and the links to PyPI have been checked to go to the right package.

What type of change are you making?
Cosmetic/ Documentation

### CODE REVIEW

**Comment by chrfoyer** [comment]
Here is the issue I am working on. I was not able to attach the PR to it.

https://github.com/vmware/versatile-data-kit/issues/1553

**Review by yonitoo — COMMENTED** [review]
Hey, @chrfoyer !
Thanks for doing this. I generally really like the idea of having such badges. 
So far, it looks good to me, left a few comments. Also, I am not sure if there is a way to propagate all these badges to the READMEs with some kind of template as all of them look like this:
<a href="https://pypistats.org/packages/<project-name>" alt="Monthly Downloads">
              .svg" alt="monthly download count for <project-name>">

We still have to find a way to not count the CI/CD-caused downloads.

File: projects/vdk-plugins/vdk-meta-jobs/README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Meta Jobs
 
+<a href="https://pypistats.org/packages/vdk-meta-jobs" alt="Monthly Downloads">
+        <img src="https://img.shields.io/pypi/dm/vdk-meta-jobs.svg" alt="monthly download count for vdk-meta-jobs">
+
```
[the remark this answers is not in the document]
Comment (chrfoyer) [reply projects/vdk-plugins/vdk-meta-jobs/README.md:5]: Yes I noticed that it was deprecated, but I figured having the badge for each can aid in showing which of them are deprecated.

File: projects/vdk-plugins/vdk-logging-format/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # This plugin has been deprecated; please use vdk-structlog instead.
 
+<a href="https://pypistats.org/packages/vdk-logging-format" alt="Monthly Downloads">
+        <img src="https://img.shields.io/pypi/dm/vdk-logging-format.svg" alt="monthly download count for vdk-logging-format">
+
```
[the remark this answers is not in the document]
Comment (chrfoyer) [reply projects/vdk-plugins/vdk-logging-format/README.md:5]: Yes I noticed that it was deprecated, but I figured having the badge for each can aid in showing which of them are deprecated.

File: specs/vep-1553-downloads-badges-in-readmes/README.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+## Summary
+
+<!--
+Short summary of the proposal. It will be used as user-focused
+documentation such as release notes or a (customer facing) development roadmap.
+The tone and content of the `Summary` section should be
+useful for a wide audience.
+-->
+Quick and accessible download statistics allow users a breif overview of how often a package is used. To deliver this information to the users, a monthly download badge has been added to each readme.
```
[the remark this answers is not in the document]
Comment (chrfoyer) [reply specs/vep-1553-downloads-badges-in-readmes/README.md:45]: Thanks! Just committed the spelling change

File: specs/vep-1553-downloads-badges-in-readmes/README.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+Specify for what type of users, what is going to be delivered.
+Recommended (but not required format) is
+User <persona/type of user>, they need <new deliverable/feature/improvement>, so that <reason>
+
+Specify non-goals. Clearly, the list of non-goals can't be exhaustive.
+Non-goals are only features, which a contributor can reasonably assume were a goal.
+One example is features that were cut during scoping.
+-->
+1. A badge with the current monthly download should be displayed in the readme for each individual package. This includes some of the larger projects, but particularly includes each plugin.
```
[the remark this answers is not in the document]
Comment (chrfoyer) [reply specs/vep-1553-downloads-badges-in-readmes/README.md:74]: Thanks! Just committed the spelling change

**Review by yonitoo — APPROVED** [review]

**Review by murphp15 — APPROVED** [review]

**Comment by murphp15** [comment]
@chrfoyer  thanks for this PR it looks brilliant and we will merge it today! 
I left just one comment on the VEP file.

File: specs/vep-1553-downloads-badges-in-readmes/README.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
+
```
Comment (murphp15) [inline specs/vep-1553-downloads-badges-in-readmes/README.md:1]: @chrfoyer Thanks for this PR is looks great!
However I think you can remove this file completely. 
VEPs need to be created when we are making more architectural changes to the platform. Where ppl will have differing opinions.

Comment (chrfoyer) [reply specs/vep-1553-downloads-badges-in-readmes/README.md]: Removed with [this commit](https://github.com/vmware/versatile-data-kit/pull/2983/commits/6dd6a7d4bbbcfca7577fa56d9ebb041a58808d14). Thanks for the feedback!

**Comment by antoniivanov** [comment]
Thanks for the contribution! I will merge it now.

**Pull request merged**